converter/value.go

```diff
@@ -48,4 +50,17 @@ type (
 		// json.Unmarshal.
 		Get(valuePtr ...interface{}) error
 	}
+
+	// RawValue is a representation of an unconverted, raw payload.
+	//
+	// This type can be used as a paramter or return type in workflows and activities to pass through
+	// a raw payload. Encoding/decoding of the payload is still done by the system.
+	RawValue struct {
```

Comment:
I wonder if we should define a `MarshalJSON` and `UnmarshalJSON` for this type that returns an error saying it is not JSON serializable.